src/cdk-experimental/column-resize/column-resize.ts

```diff
@@ -27,7 +27,7 @@ let nextId = 0;
  */
 @Directive()
 export abstract class ColumnResize implements AfterViewInit, OnDestroy {
-  protected readonly destroyed = new ReplaySubject<void>();
+  protected readonly destroyed = new Subject<void>();
```

Comment:
Is there any chance of something being destroyed before we subscribe to it? In that case the `ReplaySubject` would actually be important

Reply:
There are only internal subscriptions to this subject so it shouldn't be possible.